src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_OOP/Mammal.java

```diff
@@ -0,0 +1,68 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_OOP;
+
+public abstract class Mammal {
+    private final int NR_OF_EYES = 2;
+    private final int NR_OF_HEARTS = 1;
+    private String gender;
+    private boolean hasFur;
+    private String foodSource;
+
+    public Mammal(String gender, boolean hasFur, String foodSource) {
+        this.gender = gender;
+        this.hasFur = hasFur;
+        this.foodSource = foodSource;
+    }
+
+    public int getNR_OF_EYES() {
```

Comment:
usually we don't use underscore or other special characters for method declaration. Cant be getNumberOfEyes() or getEyesNumber()

Reply:
The final variables are written with UpperCase characters. This is why the underscores were used as well.